better_player/lib/src/video_player/video_player.dart

```diff
@@ -465,35 +461,19 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     await _videoPlayerPlatform.setLooping(_textureId, value.isLooping);
   }
 
+  //update player position
+  Future<void> updatePosition() async {
+    final Duration? newPosition = await position;
+    final DateTime? newAbsolutePosition = await absolutePosition;
+    _updatePosition(newPosition, absolutePosition: newAbsolutePosition);
+  }
+
   Future<void> _applyPlayPause() async {
```

Comment:
can remove this method as we talked? move the play/pause function to where used?